docs/reference/indices/field-usage-stats.asciidoc

```diff
@@ -0,0 +1,162 @@
+--------------------------------------------------
+GET /my-index-000001/_field_usage_stats
+--------------------------------------------------
+// TEST[setup:messages]
+
+[[field-usage-stats-api-request]]
+==== {api-request-title}
+
+`GET /<index>/_field_usage_stats
+
+[[field-usage-stats-api-request-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have the `manage`
+<<privileges-list-indices,index privilege>> for the target index or index alias.
+
+[[field-usage-stats-api-path-params]]
+==== {api-path-parms-title}
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=index]
+
+[[field-usage-stats-api-query-params]]
+==== {api-query-parms-title}
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=allow-no-indices]
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=expand-wildcards]
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=index-ignore-unavailable]
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=wait_for_active_shards]
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
+
+`fields`::
++
+--
+(Optional, string)
+Comma-separated list or wildcard expressions of fields
+to include in the statistics.
+--
+
+[[field-usage-stats-api-example]]
+==== {api-examples-title}
+
+//////////////////////////
+
+[source,console]
+--------------------------------------------------
+POST /my-index-000001/_search
+{
+  "query" : {
+    "match" : { "context" : "bar" }
+  },
+  "aggs": {
+    "message_stats": {
+      "string_stats": {
+        "field": "message.keyword",
+        "show_distribution": true  <1>
+      }
+    }
+  }
+}
+--------------------------------------------------
+// TEST[setup:messages]
+
+//////////////////////////
+
+The following request retrieves field usage information of index `my-index-000001`
+on the currently available shards.
+
+[source,console]
+--------------------------------------------------
+GET /my-index-000001/_field_usage_stats
+--------------------------------------------------
+// TEST[continued]
+
+The API returns the following response:
+
+[source,console-response]
+--------------------------------------------------
+{
+    "_shards": {
+        "total": 1,
+        "successful": 1,
+        "failed": 0
+    },
+    "my-index-000001": {
+        "shards": [
+            {
+                "tracking_id": "MpOl0QlTQ4SYYhEe6KgJoQ",
+                "tracking_started_at_millis": 1625558985010,
+                "routing": {
+                    "state": "STARTED",
+                    "primary": true,
+                    "node": "gA6KeeVzQkGURFCUyV-e8Q",
+                    "relocating_node": null
+                },
+                "stats" : {
+                    "all_fields": {
+                        "any": "6",
+                        "inverted_index": {
+                            "terms" : 1,
+                            "postings" : 1,
+                            "proximity" : 1,
```

Comment:
add a note explaining that proximity means either positions, offsets or payloads?